### DISCUSSION TITLE
자주 쓰는 색상값 추출하기

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#84] [Ideas]
@NamgyungKim 님의 디자인덕분에 이제 대표 색이랑 자주 쓰는 색들이 잡힌 거 같아요🙇🏻‍♀️ 👍
그런데 이 부분이 계속 코드에서 `#ececec` 이런식으로 반복적으로 등장하는 것보다 따로 다른 파일에 빼서 **primary color, secondary color ..**  이런 식으로 지정해주는게 좋아보이는데 다들 어떻게 생각하시나요 ??

**Comment by YuHyun-P** [comment]
아직 텍스트 스타일을 정하지 못해서 파일로 못 만들고 있는 걸로 알고 있어요~
다른 분들 속도에 맞춰야 하니 **정해진 스타일**만이라도 만드는 걸로 할까요?

**Comment by Hyevvy** [comment]
피그마에 색상이 다 나와서 지금 빼도 괜찮다고 생각했는데, 텍스트 크기 말씀하시는건가요 ?? 😮

**Reply by YuHyun-P** [reply]
네넴!

**Comment by YuHyun-P** [comment]
✅ #96
